appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
+
+import time
+import os
+import glob
+import sys
+import zipfile
+import shutil
+import subprocess
+import requests
+
+AUTH_TOKEN = os.getenv("crowdinAuthToken")
+if not AUTH_TOKEN:
+	raise ValueError("crowdinAuthToken environment variable not set")
+PROJECT_ID = int(os.getenv("crowdinProjectID"))
+if not PROJECT_ID:
+	raise ValueError("crowdinProjectID environment variable not set")
+
+
+def request(path, method=requests.get, headers={}, **kwargs):
```

Comment:
Dangerous having default mutable arguments
```suggestion
def request(path, method=requests.get, headers=None, **kwargs):
    if headers is None:
        headers = {}
```